### PR TITLE
Update the hemi example to explciit bc setting

### PR DESCRIPTION
**Pull request by timofeymukha**
- Adds boundary_types
- Add the source re2
- Add a run.sh, which launches rea2nbin and then neko..

### CODE REVIEW

**Review by njansson — COMMENTED** [review]
Not sure it is the best way to remove the nmsh file. Especially since the script is not general enough such that it can run in all environments.

I would suggest that we leave the nmsh file, but also include the re2 and the script as an example on how the mesh is generated.

**Comment by timofeymukha** [comment]
> Not sure it is the best way to remove the nmsh file. Especially since the script is not general enough such that it can run in all environments.
> 
> I would suggest that we leave the nmsh file, but also include the re2 and the script as an example on how the mesh is generated.

Yes, sounds good. I restored the nmsh

**Review by njansson — APPROVED** [review]
A suggestion would be to mention the raw re2 format in the readme, and refer users to the script for instructions on how to generate the neko mesh

File: examples/hemi/hemi.case [hunk 1]
```diff
@@ -40,6 +40,7 @@
             "absolute_tolerance": 1e-7,
             "max_iterations": 800
         },
+        "boundary_labels": ["sym", "w", "o", "v"],
```
Comment (timfelle) [inline examples/hemi/hemi.case:43]:
```suggestion
        "boundary_labels": ["sym", "w", "v", "o"],
```
It would seem you swapped the flow direction.

**Review by timfelle — CHANGES_REQUESTED** [review]

File: examples/hemi/hemi.case (hunk 1 above)
Comment (vbaconnet) [reply examples/hemi/hemi.case:43]:
```suggestion
        "boundary_types": ["sym", "w", "o", "v"],
```

This is what I get:
image

It looks okay to me, if we follow the indexing as output by `rea2nbin` no?

Comment (timfelle) [reply examples/hemi/hemi.case:43]: Well if you run the simulation on develop and on the PR the flow is reversed on
the PR. The labels might be right but the simulation is not.

Comment (timfelle) [reply examples/hemi/hemi.case:43]: But it seems the issue is fixed by your suggestion to change `boundary_labels`
to `boundary_types`

Comment (timofeymukha) [reply examples/hemi/hemi.case:43]: Ah yes, jeez. My bad, I just put the wrong keyword it seems.

**Review by timfelle — APPROVED** [review]

**Pull request merged**